src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -465,8 +465,7 @@ public void SetAccessibilityConstructor(Role role, AccessibilityInterface access
         {
```

Comment:
there is a commit message "will be removed if/when we decide to switch to AT-SPI interfaces as native C# interfaces. Then, all usages of SetAccessibilityConstructor() would be replaced by simply setting AccessibilityRole (and interface flags won't be needed)."
So this SetAccessibilityConstructor will be chaged to SetAccessibilityInterfaceAndRole or something?

Reply:
We intend to remove `SetAccessibilityConstructor` entirely! Please refer to this draft with the intended API that we want to achieve: https://github.com/Samsung/TizenFX/pull/4022.

Reply:
I got it. ```AccessibilityRole = XYZ``` will work instead of ```SetAccessibilityConstructor```